src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -365,6 +365,9 @@
         - name: --disable-disk-driver
           type: bool
           short-summary: Disable AzureDisk CSI Driver.
+        -name: azuredisk-csi-version
```

Comment:
Need space between `-` and `name`. Also, shouldn't the name be prefixed with `--` like the others options?